packages/mobile/src/consumerIncentives/ConsumerIncentivesHomeScreen.test.tsx

```diff
@@ -101,26 +105,13 @@ describe('ConsumerIncentivesHomeScreen', () => {
   }
 
   function mockQueryResponse(data: SuperchargePendingReward[]) {
-    mockFetch.resetMocks()
-    mockFetch.mockResponse(JSON.stringify({ availableRewards: data }))
+    ;(useFetchSuperchargeRewards as jest.Mock).mockImplementation(() => ({
+      superchargeRewards: data,
+      isLoading: false,
+      isError: false,
+    }))
   }
 
-  it('renders loading until the remote query finishes', async () => {
```

Comment:
Why not keeping this?

Reply:
Well, the hook is now mocked, so we would be just testing the mock in that case :/ I didn't think it was worth it to set up